packages/driver/cypress/e2e/commands/sessions/sessions.cy.js

```diff
@@ -24,6 +24,27 @@ const clearAllSavedSessions = () => {
   })
 }
 
+// In webkit, the clear page and clear cookies, etc log messages may be reversed. This isn't an issue, but we just want to test we have both messages.
```

Comment:
Do we know why this is the case? The clear page should be awaited before clearing cookies.

https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/sessions/index.ts#L360
```js
          await navigateAboutBlank()
          await sessions.clearCurrentSessionData()
```

Reply:
My assumption is that it is a timing issue with the logs and that we don't await every action in navigateAboutBlank()